geomstats/_backend/pytorch/linalg.py

```diff
@@ -20,12 +25,6 @@ def sqrtm(x):
     return torch.from_numpy(np_sqrtm)
 
 
-def expm(x):
-    np_expm = np.vectorize(
-        scipy.linalg.expm, signature='(n,m)->(n,m)')(x)
-    return torch.from_numpy(np_expm)
-
-
 def inv(*args, **kwargs):
     return torch.from_numpy(np.linalg.inv(*args, **kwargs))
```

Comment:
does `torch.inverse` work here?

Reply:
Yes! I was worried about numerical precision but it seems to be fine